README.md

```diff
@@ -22,3 +22,155 @@ e.g.
 ```
 npm i @fastify/cors -w @mykomap/back-end --save
 ```
+
+## Installation and updates
+
+Conceptually, installation of these applications require:
+- deploying the front-end as content to be served on the web
+  - configure it to use the correct path for the back end (typically `/api`)
+  - also configure API keys for GlitchTip and MapTiler services
+- deploying the back end to be run as a persistent node process
+  - configure it to use the right path to the data folder
+- deploy the data in the path expected by the back-end
+- reverse-proxying the back end to be accessible on the same domain as the front end
+
+### Deploying on a DCC server, manually
+
+DCC deployment uses a Linux server running Apache, an appropriate
+installation of NodeJS, with SystemD that supports user-mode
+services. A dedicated user is used to deploy the application, and this
+is exposed to Apache via configuration.
+
+This situation currently set up via some Ansible playbooks in the DCC
+`technology-and-infrastructure` project. The instructions following
+assume this context.
```

Comment:
This would be the Requirements subsection

Reply:
Well, again, these are not requirements of this app so much as how we happen to deploy it at the moment?

I can try and section these statements though, if that's clearer.